doc/configuration/introduction.md

```diff
@@ -21,4 +21,5 @@ p2p:
   topics_of_interest:
     messages: low
     blocks: normal
+  allow_private_addresses: true
```

Comment:
```suggestion
  allow_private_addresses: false
```

By default I would put false (especially because this is the default setting anyway). This is because under normal settings the users don't want to accept nodes that are advertising unreachable addresses.

This isn not true obviously in the context of testing locally or in a private cluster.

This will change btw, we are looking at some strategy to allow any addresses to be set anyway.

Reply:
Ok. true was selected just because the public_address has a loopback address